lib/src/editor/editor_utils.dart

```diff
@@ -428,6 +429,11 @@ class EditorConfig {
   /// default is custom
   final double? cropAspectRatio;
 
+  /// Initial Aspect ratio of crop rect
+  /// default is custom
+  /// This will appear only on init an will not persist
```

Comment:
The comment doesn't explain how the argument works.

Reply:
The argument enables us to set an initial aspect ratio on crop editor. for example you want the first things user sees to be a 1:1 crop ratio but you don't want to make user crop 1:1 always so user can change the ratio but the default ratio on screen would be ``initialCropAspectRatio``

Reply:
Yeah, it's readable from your reply, but still not so readable from the comment IMHO.